recipes-core/ssh-mode-switch/failsafe.bb

```diff
@@ -0,0 +1,7 @@
+SUMMARY = "Adds failsafe mode user and script"
+DESCRIPTION = "Creates the failsafe mode user and sets up SSH access with the provided SSH key, limit's the user to execute the failsafe.sh wrapper script"
+
+require ssh-mode-switch.inc
+
+SSH_USER_ID = "1103"
+RDEPENDS:${PN} += "curl"
```

Comment:
do we need the failsafe to be also exposed and be callable by its own SSH USER?
I thought the fail safe triggers in case the production or maintenance scripts fail.